.github/actions/security-scan-upload/action.yaml

```diff
@@ -77,7 +84,7 @@ runs:
       uses: actions/upload-artifact@v4
       if: ${{ inputs.upload-sarif-artifact == 'true' }}
       with:
-        name: 'govuln-security-scan-results'
+        name: 'govuln-security-scan-results-${{ inputs.result-suffix }}'
```

Comment:
This way, if they provide no suffix, there will be a trailing hyphen in the name, right?

Reply:
Ah yeah ...